nixui/main.py

```diff
@@ -39,6 +39,11 @@ def handle_args():
         help="Run with profiler.",
         action='store_true',
     )
+    optional.add_argument(
```

Comment:
my main concern here is that `--show-trace` is trivial for `nix` commands, but to actually get the traceback in Nix-Gui, users will need to reproduce their state after starting again with `--show-trace`

Reply:
can be solved better in a gui dialog / gui status bar message

Reply:
I think this change is risky